sqltoolbelt/tools/chocolateyinstall.ps1

```diff
@@ -35,14 +71,14 @@ if ($pp["FTP"] -ne $null -and $pp["FTP"] -ne '') {
 
 $packageArgs = @{
   packageName   = $env:ChocolateyPackageName
-  fileType      = 'exe'
-  silentArgs    = "/IAgreeToTheEula"
+  fileFullPath  = "$toolsDir\SQLToolbelt.exe"
   
-  validExitCodes= @(0)
   url           = $url
   checksum      = $checksum
   checksumType  = 'sha256'
-  destination   = $toolsDir
 }
 
-Install-ChocolateyPackage @packageArgs
+$executable = Get-ChocolateyWebFile @packageArgs
```

Comment:
I'm unsure about this bit here. Can we not keep using Install-ChocolateyPackage just passing in all the arguments in `silentArgs`?

Reply:
Oddly enough, when I originally attempted that, it didn't work. I guess there was something else wrong with the way I was running it because that appears to work now.